src/Tribe/Aggregator/Settings.php

```diff
@@ -657,7 +657,7 @@ public function get_import_limit_count_default() {
 		 *
 		 * @param int
 		 */
-		return apply_filters( 'tribe_aggregator_import_count_default', 200 );
+		return apply_filters( 'tribe_aggregator_import_count_default', 1000 );
```

Comment:
Why are we changing to 1000? It might break peoples imports, because of timeouts on the server level.

Reply:
This is a change for new installs, so users with TEC installed will remain with the current value of 200, so is backwards compatible. Reasons are described on the ticket as User Stories. 

But you might want to jump into q call with me an Geoff to discuss your concerns with the new default :) so we can find a smaller number. 

Let me know if that makes sense, thanks for your question 🙏